### ISSUE TITLE
Place suggest only a particular type

### ISSUE THREAD

**Issue by MSeguin7** [xuopled/react-google-places-suggest#26]
Hi, I am trying to made a place suggest with only a particular place type of Table 1 like restaurant for example. 
Do you have an idea how I can print in the suggest only restaurant for example.
I need to use :   type: ['restaurant']
Do you think it is possible to add it in params of the ReactGoogleMapLoader component ?
Thanks for your answer ;)

**Comment by xuopled** [comment]
Hi, following the [AutocompletionRequest object specification](https://developers.google.com/maps/documentation/javascript/reference?hl=fr#AutocompletionRequest), you can use `type` option to get only `establishment` types. It's a good start!

Here is an example:
```js
<ReactGooglePlacesSuggest
  googleMaps={googleMaps}
  autocompletionRequest={{
    input: search,
    types: ["establishment"],
  }}
/>
```

**Comment by MSeguin7** [comment]
Yeah I succeed to do that thanks. But do you have an Idea how can I select only restaurant. 
In the response we get the types of the place so I suppose I Can made something here. But I don't know how to access to the types of the response maybe something like data.types ?

**Comment by xuopled** [comment]
There is nothing in the google maps documentation for your need. I think it's not possible
